define/securityHelpers.go

```diff
@@ -84,13 +77,26 @@ func MessageIsValid(msg Message) error {
 	}
 
 	// the signature is valid
-	err := VerifySignature(msg.Sender, msg.Data, msg.Signature)
+	err := VerifySignature(msg.Sender, (msg.Data), msg.Signature)
 	if err != nil {
-		return err
+		return err //err
 	}
 
 	// the witness signatures are valid (check on every message??)
-	return VerifyWitnessSignatures()
+	publicKeys, err := AnalyseData(string(msg.Data))
+	if publicKeys["object"] == "lao" && publicKeys["action"] == "create" {
```

Comment:
`publicKeys["action"]` makes me think we're not dealing with publicKeys at all :)